toolkits/springer/packages/springer-header/js/enhanced-menu.js

```diff
@@ -0,0 +1,33 @@
+import {makeArray} from '@springernature/global-context/js/helpers';
+import {createDropdown} from '@springernature/springer-dropdown/js/create-dropdown';
+
+const createMenuDropdown = menu => {
+	menu.classList.add('u-hide-at-lt-lg');
+
+	const items = menu.querySelectorAll('li');
+	const links = makeArray(items).map(item => item.querySelector('a'));
+
+	const dropdownEl = createDropdown({
```

Comment:
just got a linting error about `dropdownEl` should be named `dropdownElement`

Reply:
ah you can see them in the travis build now..

Reply:
😭